### ISSUE TITLE
bug: Chimera Mane says to go to Wineport

### ISSUE THREAD

**Issue by phenaeri** [Supamiu/ffxiv-teamcraft#558]
Chrome version 67.0.3396.99

Have you disabled script-/adblockers before submitting?
Yes, closed all Adblockers

### Teamcraft Link
https://ffxivteamcraft.com/list/RuglPMAa9SzROnWDFMVE

## Current behavior
Clicking on "Hunting" icon for Chimera Mane describes the Allagan Chimera monster in Azys Lla however suggests to use the Wineport Aetheryte.

## Expected behavior
Chimera Mane should suggest to go to Helix Aetheryte in Azys Lla.

## Steps to reproduce
1. Create list. 
2. Populate list with items requiring Chimera Mane.
3. Click Hunting icon next on Chimera Mane row.

**Comment by Supamiu** [comment]
Related to #261 .

Should be fixed in 5.0 😄

**Comment by ayyaruq** [comment]
I know this is a known issue, and not sure exactly how related to #261, but the cause of this seems to be that they share the same aetheryte name ID in `app/core/data/sources/aetherytes.ts`. There's a bunch of other overlaps too. I think the actual map is correct, it just shows the wrong name since `nameid` is only used in the label via `this.l12n.getPlace(aetheryte.nameid)`.

Aetheryte data for Azys Lla:
````json
{
    'id': 12,
    'zoneid': 216,
    'placenameid': 2101,
    'x': 22.6,
    'y': 22.6,
    'type': 0,
    'nameid': 216
}
````

Place data for Wineport:
````json
{
    'id': 12,
    'zoneid': 17,
    'placenameid': 32,
    'x': 21.1,
    'y': 21.6,
    'type': 0,
    'nameid': 216
}
````
